src/wwwroot/env/template-editor/INT.js

```diff
@@ -0,0 +1,5 @@
+﻿angular.module('mseditor').constant('RELAY_CONFIG', {
```

Comment:
Is it possible using the same configuration files for WebApp and templates editor?

Reply:
I don't think so but, maybe we can move the environment related files to a env folder inseide template editor due to template editor is separated of webapp and handle the bundle from the gulpfile mergin js files inside template editor folder.